api/types/Thread.js

```diff
@@ -54,6 +54,7 @@ const Thread = /* GraphQL */ `
     attachments: [Attachment]
     watercooler: Boolean
     currentUserLastSeen: Date @cost(complexity: 1)
+    reactions: ReactionData @cost(complexity: 1)
```

Comment:
I'm going to veto using the same schema as the message reactions because it doesn't let us show which users liked the thing. (you only see it in the notifications, but nobody else can see/check) Let's change the return type here to include that information somehow, (open to ideas) so we can in the future show who liked a thread or not in the UI.

Reply:
Going to change this to be a separate type `ThreadReactions` which will expose:

`
count: Int
hasReacted: Boolean
reactors: [User]
`

Or something like that - we can actually extend the message reactions to have the same data model, but let's keep message reactions and thread reactions as separately defined types in case they diverge in the future.